tests/reporter/kubernetes.go

```diff
@@ -302,6 +303,56 @@ func (r *KubernetesReporter) logAuditLogs(virtCli kubecli.KubevirtClient, since
 	}
 }
 
+func (r *KubernetesReporter) logJournal(virtCli kubecli.KubevirtClient, duration time.Duration) {
+
+	const component = "journal"
+
+	logsdir := filepath.Join(r.artifactsDir, "nodes")
+	if err := os.MkdirAll(logsdir, 0777); err != nil {
+		fmt.Fprintf(os.Stderr, "failed to create directory %s: %v\n", logsdir, err)
+		return
+	}
+
+	logDuration := strconv.FormatInt(int64(duration/time.Second), 10)
+
+	nodes := getNodesWithVirtLauncher(virtCli)
+
+	for _, node := range nodes {
+		pod, err := kubecli.NewVirtHandlerClient(virtCli).Namespace(tests.KubeVirtInstallNamespace).ForNode(node).Pod()
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "failed to get virt-handler pod on node %s: %v", node, err)
+			return
+		}
+
+		commands := []string{
+			"/usr/bin/virt-chroot",
+			"--mount",
+			"/proc/1/ns/mnt",
+			"exec",
+			"--",
+			"/usr/bin/journalctl",
+			"--since",
+			"-" + logDuration + "s",
+		}
+		stdout, stderr, err := tests.ExecuteCommandOnPodV2(virtCli, pod, "virt-handler", commands)
+		if err != nil {
+			fmt.Fprintf(
+				os.Stderr,
+				"failed to execute command: %s, stdout: %s, stderr: %s, error: %v",
+				commands, stdout, stderr, err,
+			)
+			return
+		}
+
+		fileName := fmt.Sprintf("%d_%s_%s.log", r.failureCount, component, node)
```

Comment:
i would suggest please to switch node and component, so in case of couple components, file list would be sorted according node

Reply:
This is the current pattern used in all the other node level collections, it will be wrong to change this in only one place.